js/samples/advanced-csharp/Pages/Options.cshtml

```diff
@@ -0,0 +1,131 @@
+﻿@page
+@model AdvancedSampleWebApp.Pages.OptionsModel
+
+<!doctype html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <title>Immersive Reader Example: SDK options</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+
+    <script type='text/javascript' src='https://code.jquery.com/jquery-3.3.1.min.js'></script>
+    <script type='text/javascript' src='https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js'></script>
+    <script type='text/javascript' src='https://ircdname.azureedge.net/immersivereadersdk/immersive-reader-sdk.1.1.0.js'></script>
+    <script type='text/javascript' src='~/js/helpers.js'></script>
+
+    <link href='~/css/styles.css' rel='stylesheet'>
+    <link href='~/css/buttons.css' rel='stylesheet'>
+
+    <script type='text/javascript'>
+        var Canary = '@Html.Raw(ViewData["Canary"])';
+    </script>
+
+    <style media='screen' type='text/css'>
+        #ContentArea {
+            display: block;
+            margin: 0 20%;
+            padding-top: 50px;
+            width: 60%;
+        }
+
+        .section {
+            border: 1px solid #cccccc;
+            border-radius: 4px;
+            box-sizing: border-box;
+            display: block;
+            margin-bottom: 20px;
+            padding: 20px 20px 10px 20px;
+            position: relative;
+            width: 100%;
+        }
+
+        #Cols {
+            box-sizing: inherit;
+            column-count: 1;
+            column-gap: 0;
+        }
+
+        h3 {
+            margin: 0;
+            margin-top: 10px;
+            padding: 0;
+        }
+
+        p {
+            line-height: 25px;
+        }
+
+        @@media screen and (min-width: 900px) {
+            #Cols {
+                column-gap: 20px;
+                column-count: 2;
+            }
+        }
+    </style>
+</head>
+<body class='body'>
+    <nav>
+        Examples:
+        <a href='/index'>Sections</a>
+        <a href='/document'>Document</a>
+        <a href='/multilang'>Multilingual Document</a>
+        <a href='/uilangs'>UI Language</a>
+        <a href='/math'>Math</a>
+        <a href='/worddoc'>Word Document</a>
+        <a href='/options' class='active'>Options</a>
```

Comment:
Is this the structure you are planning for other options as well? (when you add them later)

Reply:
@rwallerms 
I can add individual examples in this same page, but we could have lot of options at any point.
So, then are you suggesting having a single page for each option?

Reply:
yes, maybe, but not now

Reply:
Thank you, I will consider that